webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
```

Comment:
p.then(fulfillment handler, undefined) is not correct because what happens if some script on the page does `Promise.prototype.then = function () { throw new Error("No, I am your father."); }`.

I can change the heading of the document to "TAG Finding" if that would help.